src/storage-controller/src/lib.rs

```diff
@@ -1113,6 +1115,7 @@ where
                     status_id,
                     from_storage_metadata,
                 },
+                update: false,
```

Comment:
Why does the cluster need to know that this is an update? Can't it just check if the description it currently has matches the description it received?

Reply:
This is used in storage state assertions to express whether or not the workers should expect this to be an update to an existing description.

Reply:
Will resolve this feedback as part of [this comment](https://github.com/MaterializeInc/materialize/pull/21109#discussion_r1395496260)